dask/delayed.py

```diff
@@ -246,28 +249,68 @@ def _inner(self, other):
     return _inner
 
 
+class DelayedFunction(base.Base):
+    _optimize = staticmethod(lambda dsk, keys, **kwargs: dsk)
+    _finalize = staticmethod(first)
+    _default_get = staticmethod(get_sync)
```

Comment:
The way the generic `compute` works, `compute(delayed_function, delayed)` will barf, because they define two different schedulers as the default.